### ISSUE TITLE
ndb doesn't work when debugging gitbook-cli 

### ISSUE THREAD

**Issue by cctv1005s** [GoogleChromeLabs/ndb#66]
### Steps to reproduce

**Tell us about your environment:**

* ndb version:  1.0.6
* Platform / OS version: macOS High Sierra 10.13.6 17G65 x86_64 
* Node.js version: v10.5.0

**What steps will reproduce the problem?**

_Please include code that reproduces the issue._

1. vim /usr/local/lib/node_modules/gitbook-cli/bin/gitbook.js
2. insert debugger in the front of code
3. ndb node /usr/local/lib/node_modules/gitbook-cli/bin/gitbook.js 

**What is the expected result?**
program stops at debugger;

**What happens instead?**
program goes, ignores debugger

gitbook.js after edit

![image](https://user-images.githubusercontent.com/16508727/43308362-f238e8f8-91b3-11e8-8797-505e589c94e2.png)

program result:
![image](https://user-images.githubusercontent.com/16508727/43308462-3f3eb934-91b4-11e8-8c0b-379c51430501.png)

devtools: 
![image](https://user-images.githubusercontent.com/16508727/43308476-4c859360-91b4-11e8-86f4-1bedd0c24272.png)

**Comment by ak239** [comment]
Thanks for you report!
Could you try latest ndb version? Currently it is 1.0.13.

**Comment by ak239** [comment]
You can go to settings and disable "Blackbox anything outside working directory" checkbox, or you can try to go to the folder with gitbook-cli and run it from there:
```bash
cd /usr/local/lib/node_modules/gitbook-cli/
ndb ./bin/gitbook.js
```
We will improve this use case by better detecting it and auto-disabling blackboxing feature.

**Comment by cctv1005s** [comment]
@ak239 thank you, i will try soon. 😘

**Comment by ak239** [comment]
Issue with the same root: https://github.com/GoogleChromeLabs/ndb/issues/151 .

**Comment by ak239** [comment]
This issue should be fixed in ndb@1.0.37.
